lib/navigation/consts.hpp

```diff
@@ -0,0 +1,46 @@
+#include <cstdint>
+
+
+namespace hyped::navigation{
```

Comment:
formatting

Reply:
yeah formatting is on the to-do list. Will make sure it's done before making this pr ready for review

Reply:
Pro-tip: Run `$ ./scripts/setup` to get formatting on commits.